### ISSUE TITLE
[UI] Explore page : images of any format in square boxes

### ISSUE THREAD

**Issue by joachimesque** [pixelfed/pixelfed#245]
In order to display the Explore page’s image grid in equally-sized blocks, like in this proposition :

![screenshot-2018-6-9 pixelfed](https://user-images.githubusercontent.com/1781070/41193184-41f44598-6c09-11e8-92d0-83a00bc04c9b.png)

I suggest you remove the `height` and add an `object-fit: contain` directive.

```
.card-img-top {
--    height: auto;
++    object-fit: contain;
}
```

`object-fit` on images is compatible with all modern browsers https://caniuse.com/#search=object-fit

**Comment by hellcp** [comment]
Fixed in #168

**Comment by joachimesque** [comment]
Oh, I hadn't seen the branch `frontend-ui-refactor`, most of my issues should be closed now 😓
